internal/util/logging/console_handler_test.go

```diff
@@ -49,3 +54,96 @@ func TestConsoleHandler(t *testing.T) {
 
 	slogtest.Run(t, newHandler, result)
 }
+
+func TestConsoleHandlerEscapeCodes(t *testing.T) {
+	t.Parallel()
+
+	var buf bytes.Buffer
+	mockTerm := term.NewTerminal(&buf, "")
+
+	ch := &consoleHandler{
+		opts: &NewHandlerOpts{Level: slog.LevelInfo},
+		m:    new(sync.Mutex),
+		out:  mockTerm,
+		esc:  mockTerm.Escape,
+	}
+
+	for name, tc := range map[string]struct {
+		expected string
+		level    slog.Level
+	}{
+		"Debug-2": {
+			level:    slog.LevelDebug - 2,
+			expected: "\033[34mDEBUG-2\033[0m\tfoobar\r\n",
+		},
+		"Debug-1": {
+			level:    slog.LevelDebug - 1,
+			expected: "\033[34mDEBUG-1\033[0m\tfoobar\r\n",
+		},
+		"Debug": {
+			level:    slog.LevelDebug,
+			expected: "\033[34mDEBUG\033[0m\tfoobar\r\n",
+		},
+		"Debug+1": {
+			level:    slog.LevelDebug + 1,
+			expected: "\033[34mDEBUG+1\033[0m\tfoobar\r\n",
+		},
+		"Debug+2": {
+			level:    slog.LevelDebug + 2,
+			expected: "\033[34mDEBUG+2\033[0m\tfoobar\r\n",
+		},
+		"Info": {
+			level:    slog.LevelInfo,
+			expected: "\033[32mINFO\033[0m\tfoobar\r\n",
+		},
+		"Warn": {
+			level:    slog.LevelWarn,
+			expected: "\033[33mWARN\033[0m\tfoobar\r\n",
+		},
+		"Error": {
+			level:    slog.LevelError,
+			expected: "\033[31mERROR\033[0m\tfoobar\r\n",
+		},
```

Comment:
You even wrote tests that assume that there are no log levels between, say, warn and error – but there are